README.md

```diff
@@ -88,10 +88,11 @@ Here are examples:
 * Homebrew - [Holmes, E.: "How i gained commit access to homebrew in 30 minutes", 2018](https://medium.com/@vesirin/how-i-gained-commit-access-to-homebrew-in-30-minutes-2ae314df03ab)
 * Gentoo Linux - [Khandelwal, S. "Password-guessing was used to hack gentoo linux github account", 2017]( https://thehackernews.com/2018/07/github-hacking-gentoo-linux.html)
 
-MFA tokens don't counter all attacks (such as typosquatting), but by using tools such
-as Multi-factor Authentication, the likelihood that such bad actors 
+MFA tokens don't counter all attacks (such as typosquatting and also do not leave your token unguarded in public or not trusted spaces as there are known [side-channel attacks](https://www.zdnet.com/article/new-side-channel-attack-can-recover-encryption-keys-from-google-titan-security-keys/) for hardware tokens.), but by using tools such as Multi-factor Authentication, the likelihood that such bad actors 
```

Comment:
"typosquatting" should be "typosquatting)".

This is a run-on sentence, making it hard to understand. Replace "and also" with ". Also," and reword the rest.

Reply:
I have adjusted it